client/app/bundles/ActionHeader/components/Index.jsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-clinical-action-header/docs/README.md';
+import { version } from 'terra-clinical-action-header/package.json';
+
+// Example Files
+import ActionHeaderBackNextPreviousClose from 'terra-clinical-site/lib/examples/action-header/ActionHeader-BackNextPreviousClose';
+import ActionHeaderExpandClose from 'terra-clinical-site/lib/examples/action-header/ActionHeader-MaximizeClose';
+import ActionHeaderMinimizeCustomClose from 'terra-clinical-site/lib/examples/action-header/ActionHeader-MinimizeCustomClose';
+
+const HeaderExamples = () => (
+  <div>
+    <div id="version">Version: {version}</div>
+    <Markdown id="readme" src={ReadMe} />
+    <h2>Action Header</h2>
+    <h2>Back, Previous-Next, Close</h2>
+    <ActionHeaderBackNextPreviousClose />
+    <h2>Maximize, Close</h2>
+    <ActionHeaderExpandClose />
+    <h2>Minimize, Custom Content, Close </h2>
+    <ActionHeaderMinimizeCustomClose />
+  </div>
+);
+
+export default HeaderExamples;
```

Comment:
That will fix the missing translation, however it is falling back to the id as the message. Is that the behavior we want? 



![screen shot 2017-06-21 at 9 26 02 am](https://user-images.githubusercontent.com/8146159/27389332-b1c0c368-5663-11e7-885c-4f3d779f9f6b.png)

![screen shot 2017-06-21 at 9 26 33 am](https://user-images.githubusercontent.com/8146159/27389357-bfeb54da-5663-11e7-8b6b-40cd33bb55dd.png)
```
[React Intl] Cannot format message: "Terra.Clinical.ActionHeader.previous", using message id as fallback.
```